### DISCUSSION TITLE
How to wrap the RowData component?

### DISCUSSION THREAD

**Discussion by gerryfletch** [komarovalexander/ka-table#249] [Q&A] [answered]
Hi,

I would like to keep the row data component, but wrap the entire row with an `<a href="...">{row}</a>`. I can override the component in its entirety, but I get memory issues invoking `<RowData {...props}/>` directly. Is there an alternative way to do this?

**Comment by komarovalexander** [comment, marked as answer]
Hi @gerryfletch row it is a `<tr>` element inside `<tbody>`, I don't think this is the right way to place `<a>` inside `tbody`. I would suggest to add onClick event to row element instead, here is example how to add onClick to row https://komarovalexander.github.io/ka-table/#/events

**Reply by gerryfletch** [reply]
Ah! Nice, thank you
